packages/neo-one-node-vm/src/__tests__/syscalls.test.ts

```diff
@@ -137,6 +173,8 @@ interface TestCase {
   readonly options?: Options;
   // tslint:disable-next-line no-any
   readonly mock?: ((options: { readonly blockchain: any }) => void);
+  readonly error?: string;
+  readonly flags?: Set<string>;
```

Comment:
Make the flags a specific type, e.g. `type Flag = 'noPersistingBlock' | 'someOtherFlag'` and here `Set<Flag>`